governance/votes/Executive Vote - April 30, 2025.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Template - [Executive Vote] STAR2 Allocation System Updates, Increase GSM Pause Delay, Add Emergency Spell to Chainlog, Top-up of the Integration Boost, Launch Project Funding, Spark Proxy Spell, STAR2 Proxy Spell - April 30, 2025
+summary: Update STAR2 Allocation Parameters for Technical Test, Increase the GSM Pause Delay, Add EMSP_SPBEAM_HALT to the Chainlog, Transfer funds for Integration Boost, Transfer funds for Launch Project, Execute Spark proxy Spell, Execute STAR2 proxy spell.
+
+date: 2025-04-30T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] STAR2 Allocation System Updates, Increase GSM Pause Delay, Add Emergency Spell to Chainlog, Top-up of the Integration Boost, Launch Project Funding, Spark Proxy Spell, STAR2 Proxy Spell - April 30, 2025
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- STAR2 Allocation System will be updated as described below.
+- The [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055) will be increased.
+- EMSP_SPBEAM_HALT Standby Spell at [0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0](https://etherscan.io/address/0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0) will be added to the Chainlog.
```

Comment:
```suggestion
- EMSP_SPBEAM_HALT Standby Spell at [0xDECF4A7E4b9CAa3c3751D163866941a888618Ac0](https://etherscan.io/address/0xDECF4A7E4b9CAa3c3751D163866941a888618Ac0) will be added to the [Chainlog](https://chainlog.sky.money).
```

This suggestion includes VW's suggestion (you can only apply one suggestion per line)